docs/guides/layout.md

```diff
@@ -0,0 +1,101 @@
+# Layout
+
+Video.js generally lays out the player to the dimensions that are set as attributes or via CSS, like other DOM elements. However, we provide a few ways to make the player be more fluid.
+
+## Fluid mode
+
+Video.js has a fluid mode that keeps the player sized to a particular aspect ratio.
+By default, fluid mode will use the intrinsic size of the video once loaded but you can change it with classes or with the `aspectRatio` option.
+Enabling fluid mode will disable fill mode. If both are enabled, fluid mode takes precedence, currently.
+
+You can enable fluid one in a few ways:
+
+- Add `vjs-fluid`, `vjs-16-9`, or `vjs-4-3` as a class to the player element.
+- pass `fluid` option to the player.
+- call `player.fluid(true)`.
+- pass `aspectRatio` option to the player.
+- call `player.aspectRatio('16:9)`.
+
+### Classes
+
+There are three classes associated with fluid mode, `vjs-fluid`, `vjs-16-9`, and `vjs-4-3`.
+`vjs-fluid` turns on the general fluid mode which will wait for the video to load to calculate the aspect ratio of the video.
+Alternatively, because 16:9 and 4:3 aspect ratios are so common, we provided them as classes by default for you to use if you know that your videos are 16:9 or 4:3.
+
+### fluid option
+
+You can pass in the `fluid` option to the player or call `player.fluid(true)`. This will enable the generic fluid mode.
```

Comment:
I think it would make sense to move rename this section `enabling fluid mode` and move the list from the section above down here.